### ISSUE TITLE
[Linux] Cannot map inputs to LT and RT on Logitech F710

### ISSUE THREAD

**Issue by jpd002** [jpd002/Play-#782]
Originally reported here #562 by @ SergeyLatyshev.

@Zer0xFF does this ring any bell to you?

**Comment by Zer0xFF** [comment]
No, it was never an issue for me, I'll take a look at the code and see what the issue might be. I recall on one of the platforms (probably Linux) I had to filter out the gyroscope reading from DS4 to be able to assign buttons, might be a false positive.
